Makefile

```diff
@@ -46,3 +45,38 @@ python-bootstrap: venv
 
 venv:
 	test -d venv || virtualenv venv
+
+memorydiskserver-docker-image:
+	mkdir -p memorydiskserver-build
+	CGO_ENABLED=0 GOOS=linux godep go build -a -ldflags '-s' cmd/memorydiskserver/memorydiskserver.go
+	mv memorydiskserver memorydiskserver-build/
+	cp Dockerfile.memorydiskserver memorydiskserver-build/Dockerfile
+	cd memorydiskserver-build && docker build -t clusterhq/memorydiskserver .
+
+dvol-golang-docker-image:
```

Comment:
I've attempted to address this in https://github.com/ClusterHQ/dvol/blob/docker-interacting-dvol-tests-FLOC-4258/scripts/build-golang-docker-image.sh